bin/runner.js

```diff
@@ -168,6 +168,12 @@ var argv = yargs
           type: "string",
           default: undefined
         },
+        't': {
```

Comment:
Need to add help section for this arg. Also specify `mins` when giving range in messgae

Reply:
by help section do you mean the description have added it under `SPEC_TIMEOUT`